requirements/default.txt

```diff
@@ -319,3 +319,7 @@ wheel==0.29.0 \
 whitenoise==3.3.1 \
     --hash=sha256:15f43b2e701821b95c9016cf469d29e2a546cb1c7dead584ba82c36f843995cf \
     --hash=sha256:9d81515f2b5b27051910996e1e860b1332e354d9e7bcf30c98f21dcb6713e0dd
+
+# A Python library for Stripe’s API.
```

Comment:
This should include some more data:

```
# Code: https://github.com/stripe/stripe-python
# Changes: https://github.com/stripe/stripe-python/blob/master/CHANGELOG.md
# Docs: https://stripe.com/docs/api/python#intro
```

Since you started, stripe 2.8.0 was released.